src/test/java/plugins/AbstractAnalysisTest.java

```diff
@@ -171,6 +171,17 @@ public void should_send_mail_with_expanded_tokens() {
                 String.format("%s: %d-0-%d", title, getNumberOfWarnings(), getNumberOfWarnings()));
     }
 
+
+    /**
+     * Restart Jenkins if there is not required extension - see JENKINS-49026
+     */
+    protected void checkExtensionAreDeployed(String extensionName, String extensionPointName){
+        String result = jenkins.runScript("Jenkins.instance.getExtensionList(" + extensionPointName + ".class)");
```

Comment:
Is ".class" required? When I run thin in the script console I get an exception!

Reply:
it works both for me, so no problem to change it, if it causes problem with .class.